packages/integration-tests/test/actions.spec.ts

```diff
@@ -44,18 +51,25 @@ const initWatcher = async () => {
 }
 
 const deploySimpleStorage = async () => {
-  const SimpleStorageJson = JSON.parse(fs.readFileSync('contracts/build/SimpleStorage.json').toString())
-  const SimpleStorageFactory = new ContractFactory(SimpleStorageJson.abi, SimpleStorageJson.bytecode, l2Wallet)
-  const dummy = await SimpleStorageFactory.deploy()
-  await dummy.deployTransaction.wait()
-  return SimpleStorageFactory.deploy()
+  const simpleStorageJson = JSON.parse(fs.readFileSync('contracts/build/SimpleStorage.json').toString())
```

Comment:
This could be a `require` instead of `fs.readFileSync`, `require` on a json file will turn it into an object automatically

Reply:
Nice. Definitely cleaner